packages/automerge-repo-demo-counter-svelte/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "automerge-repo-demo-counter-svelte",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json"
+  },
+  "dependencies": {
+    "svelte": "^3.49.0",
+    "@automerge/automerge": "^2.0.3",
+    "automerge-repo": "^0.0.50",
+    "automerge-repo-network-broadcastchannel": "^0.0.50",
+    "automerge-repo-network-websocket": "^0.0.50",
+    "automerge-repo-storage-localforage": "^0.0.50",
+    "automerge-repo-svelte-store": "./packages/automerge-repo-svelte-store"
```

Comment:
reminder: this should be a version number

Reply:
You should just be able to replace this with a version number and it works. If not, try also running `lerna link --force-local` from the repo root.